src/app/containers/GetDomainStateContainer.js

```diff
@@ -5,7 +5,7 @@ import { GetDomainStateComponent } from '../components';
 
 const mapStateToProps = state => ({
   domain: parse(state.router.location.search).domain,
-  disableSearchButton: state.registrar.committing || state.registrar.committed,
+  disableSearchButton: (state.registrar.committing || state.registrar.committed) && state.router.location.pathname === '/registrar',
```

Comment:
Ahora si 😁

Reply:
I think we could add some tests over here. I’ll approve anyway and open a new issue about testing some key containers

Reply:
#71